tests/backup/backup_basic_test.go

```diff
@@ -44,6 +44,7 @@ var (
 		"cassandra": {"pre_action_list": "nodetool flush -- keyspace1;", "post_action_list": "nodetool verify -- keyspace1;", "background": "false", "run_in_single_pod": "false"},
 		"postgres":  {"pre_action_list": "PGPASSWORD=$POSTGRES_PASSWORD; psql -U '$POSTGRES_USER' -c 'CHECKPOINT';", "background": "false", "run_in_single_pod": "false"},
 	}
+	backup_crd_list = []string{"backuplocations.stork.libopenstorage.org", "applicationbackups.stork.libopenstorage.org", "applicationbackupschedules.stork.libopenstorage.org"}
```

Comment:
This check has to be done on the application cluster and not on the backup cluster, in this case we might want to run on both the cases.  Additionally we would want to validate these crds as well: 
```
applicationrestores                                                                                  stork.libopenstorage.org/v1alpha1           true         ApplicationRestore
schedulepolicies                                                                                     stork.libopenstorage.org/v1alpha1           false        SchedulePolicy
rules                                                                                                stork.libopenstorage.org/v1alpha1           true         Rule
```

Reply:
Done!!